.gitignore

```
@@ -23,3 +22,6 @@ automation/ocr/ocrconfig.meta
 automation/deltaCalculator.log
 automation/ocr/ocrconfig.meta
 automation/ocr/ocrconfig.meta
+
+venv/
+__pycache__
```

*.txt gets ignored. So I think we have to explicitly include requirements.txt
```suggestion
__pycache__
!requirements.txt
```